docs/blog/posts/bazzite-buzz-05.md

```diff
@@ -0,0 +1,94 @@
+---
+date: 2023-10-04
+comments: true
+authors: 
+  - nicknamenamenick
+links:
+  - Bazzite Commits: https://github.com/ublue-os/bazzite/commits/main
+  - Bazzite 1.0: https://universal-blue.org/blog/2023/08/20/bazzite-10/
+---
+![](https://hackmd.io/_uploads/ryvAQvC16.jpg)
+<hr>
+
+<p style="text-align: center; font-weight: 600; color: red">IMPORTANT: MANUAL INTERVENTION REQUIRED</p>
+
+If you installed a Bazzite image on any device **before September 22, 2023** or used the **older offline ISO** then open a host terminal and enter `rpm-ostree update`, and wait for it to upgrade to the latest build.  Reboot your device after it has finished.  More information on this [here](https://github.com/ublue-os/bazzite/issues/382).
+<hr>
+
+<p style="font-weight:900; font-size: 25px">Welcome to Bazzite Buzz #5!</p>
+
+Bazzite is a custom image of Fedora 38 designed to being the best in Linux gaming to your PCs, including the Steam Deck. This newsletter highlights all the work we have been doing to bring gamers the best goodies for their PCs and handheld gaming devices.
+
+If you are new to a Universal Blue project then here's how they work then here's how they work. They follow the [continuous delivery](https://continuousdelivery.com/) methodology of development. That means we're constantly adding new things, so let's get started!
+
+We have been preparing for the stable release of Fedora 38 and will plan to switch builds over there within the next two weeks.  Nvidia images for ASUS and Microsoft Surface hardware is now available.  We also added dependencies needed for the [DaVinci Resolve](https://www.blackmagicdesign.com/products/davinciresolve) video editor in case you wanted that.  The [`unrar`](https://github.com/aawc/unrar) library is included in the image which should resolve issues users may experience with certain compressed file formats.
```

Comment:
```suggestion
We have been preparing for the stable release of Fedora 39 and will plan to switch builds over there within the next two weeks.  Nvidia images for ASUS and Microsoft Surface hardware is now available.  We also added dependencies needed for the [DaVinci Resolve](https://www.blackmagicdesign.com/products/davinciresolve) video editor in case you wanted that.  The [`unrar`](https://github.com/aawc/unrar) library is included in the image which should resolve issues users may experience with certain compressed file formats.
```
I think you mean 39?